content/client-server-api/_index.md

```diff
@@ -2837,7 +2837,7 @@ most recent message events for each room, as well as the state of the
 room at the start of the returned timeline. The response also includes a
 `next_batch` field, which should be used as the value of the `since`
 parameter in the next call to `/sync`. Finally, the response includes,
-for each room, a `prev_batch` field, which can be passed as a `start`
```

Comment:
`start` doesn't existing for [`/messages`](https://spec.matrix.org/v1.14/client-server-api/#get_matrixclientv3roomsroomidmessages). It's `from`/`to` with a `dir`